tests/test_table_schema.py

```diff
@@ -0,0 +1,15 @@
+# pylint: disable=missing-module-docstring, missing-function-docstring
+import pytest
+from pydantic import ValidationError
+
+from dynamicio.config.pydantic.table_schema import SchemaColumn
+
+
+def test_schema_column_accepts_valid_pandas_types():
+    column = SchemaColumn(name="test", type="int64")
+    assert column.data_type == "int64"
```

Comment:
Worth parameterising this one to make it exhaustive of all pandas types? (I admit it feels redundant; up to you)

Reply:
it's actually surprisingly difficult to get a list - all the relevant interfaces are private (also I think it depends on whether you have stuff like geopandas installed)